src/index.js

```diff
@@ -90,7 +98,7 @@ export function app(state, actions, view, container) {
   }
 
   function get(path, source) {
-    for (var i = 0; i < path.length; i++) {
+    for (var i in path) {
```

Comment:
Why are you touching this? @etalisoft

Reply:
@JorgeBucaran 
Given the focus on the end product file size, I was looking for ways to make up for some of the bytes that I added.  Obviously this can get reverted as it has no impact on the Connected Component code.

```js
var arr1 = ['a'];
arr1[100] = 'b';
for (var i = 0; i < arr1.length; i++) 
	console.log('arr1', i);
// outputs 100 times
  
var arr2 = ['a'];
arr2[100] = 'b';
for (var i in arr2)
	console.log('arr2', i);
// outputs twice
```

Based on the above logic, in addition to savings file size, it seemed it could have the added benefit of possibly reducing iterations.

Reply:
Let's implement it first, then worry about file size. Please revert it.

Reply:
```js
arr2[100] = 'b';
```
Haha yeah, but that **can't** happen and regular for loops are faster than slow foreach loops.